packages/atomic/template/src/pages/index.html

```diff
@@ -4,12 +4,17 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>{{titleCase project}}</title>
-    <script
-      type="module"
-      src="https://static.cloud.coveo.com/atomic/v1/atomic.esm.js"
-    ></script>
-    <link href="/build/app.css" rel="stylesheet" />
-    <script type="module" src="/build/app.esm.js"></script>
+    <link href="/build/{{project}}.css" rel="stylesheet" />
+    <!-- 
+      When using the Coveo CDN - Make sure you're using the same minor version as the packaged Atomic.
+      E.g., if you have "@coveo/atomic@1.44.3" installed, use the "/atomic/v1.44/" path.
+    
+      <script type="module" src="https://static.cloud.coveo.com/atomic/v1.MINOR/atomic.esm.js"></script> 
+    -->
+    <script type="module" src="/atomic/atomic.esm.js"></script>
+    <!-- Custom Elements Lazy Loading -->
+    <script type="module" src="/build/{{project}}.esm.js"></script>
+    <!-- Global initialization script -->
```

Comment:
I think we could do both:
index.html can stay like this
index-cdn.html for the CDN-ed version?

Reply:
I don't want the users to maintain 2 html pages, I'd rather just have this comment and have them make the decision early and stick to it. The CDN is more risky when building custom components, I'd rather push for a better dev setup OOTB.

The index.html is also the one that's loaded when starting the dev server, users might be confused?

Reply:
mh okok, looks good enough for now.
One thing that we might wanna check later on is how to help user go from their development environment to prod.
I reckon that a user that deploy its website should use the CDN in its deployment for optimum perfs, correct?

Reply:
> One thing that we might wanna check later on is how to help user go from their development environment to prod.

Yes it's the phase 3!

> I reckon that a user that deploy its website should use the CDN in its deployment for optimum perfs, correct?

Yes for prod not having to serve these files is better from the customer's perspective